pegasus/scan_test.go

```diff
@@ -453,3 +453,51 @@ func TestPegasusTableConnector_ScanOverallScan(t *testing.T) {
 
 	compareAll(t, dataMap, baseMap)
 }
+
+func setDatabase2(tb TableConnector) {
```

Comment:
I think 'setDataBase' is ok, no need to add a postfix '2'

Reply:
There is another function named `setDataBase`